### ISSUE TITLE
[LS] Record field renaming doesn't work for parameters in function invocations

### ISSUE THREAD

**Issue by IMS94** [ballerina-platform/ballerina-lang#30935]
**Description:**
Consider the following code:
```ballerina
public function main() {
    doSomething({id: 9, 'type: models:DOG, name: "Name", breed: "Unknown"});
}

function doSomething(models:Pet p) {
}
```
and `Pet` record is as follows:
```ballerina
public type Pet record {
    int id;
    Type 'type;
    string name;
    string breed;
};
```
If we rename `id` field in `Pet` record, the field `id` in `doSomething()` invocation is not renamed.

**Steps to reproduce:**
See description

**Affected Versions:**
SL Beta 1 RC8

**Comment by pubudu91** [comment]
Isn't this a new feature? From semantic API side, finding refs is only supported within a module. IIRC, doing it across modules required some additional capabilities from the project API side.

**Comment by IMS94** [comment]
@pubudu91 At LS side, we look for references to a symbol across all modules. See `ReferencesUtil.getReferences()`. Even if we have the record type def and the usage in the same file, rename/references doesn't work.

**Comment by IMS94** [comment]
Relates to #22009

**Comment by IMS94** [comment]
This should be fixed automatically by #22009. Marking this as blocked until that issue is fixed.

**Comment by IMS94** [comment]
Checked on SL Beta 3 Snapshot and this has been fixed when resolving #22009.
